shared/css/sidebar.css

```diff
@@ -3,96 +3,502 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 /* XXX akita: Tidy up css */
-.media-wrapper:not(.showing-local-streams) > .media-control-buttons {
+/*------------------------------------*\
+    SIDEBAR
+\*------------------------------------*/
+
+body.sidebar {
+  height: 100%;
+}
+
+.sidebar-wrapper {
+  display: flex;
+  flex-flow: column wrap;
+  width: 250px;
+  height: 100%;
+}
+
+.sidebar-wrapper > .media-layout {
+  /* 188px video height + 40px media buttons height */
+  height: calc(188px + 40px);
+}
+
+.sidebar-wrapper > .media-layout > .media-wrapper {
```

Comment:
This feels a bit strange, reaching down into the wrapper, when you're not doing that elsewhere.